src/renderer/store/me.store.ts

```diff
@@ -17,4 +17,19 @@ export const _me = reactive({
     status: "offline",
 } as CurrentUser);
 
+async function login() {
+    await window.auth.login();
+    _me.isOnline = true;
```

Comment:
not quite true, because you still need to actually login and open the websocket connection. That can fail for a number of reasons, you're banned for example.
You can leave that in but keep in mind getting an auth token is not the same as actually logging in